distribution/docs/src/main/resources/ManContents.adoc

```diff
@@ -221,6 +222,27 @@ Complete the following procedure now that several queries have been executed.
 Based on the browser's configuration, the .xls file will be downloaded or automatically displayed in Excel.
 ====
 
+=== Configuration Export/Import
+==== Export Configuration Settings
+
+1) Launch DDF.
+
+2) At the Admin Console, select any Active Application that you want to save.
+
+3) For each Active Application selected, click Save changes on the configuration that you want to save. This creates the configuration files.
+
+4) Using the DDF command line interface (CLI), type in config-export. The exported files will be exported to <DISTRIBUTION_INSTALL_DIR>/etc/exported
```

Comment:
Beyond UI and CLI, nothing standardized. It probably would be a good idea, though.

Yes to back-ticks for monospace formatting.